README.md

```diff
@@ -1,25 +1,28 @@
 Bitcoin XT
 ==========
 
-Bitcoin XT is an implementation of a Bitcoin full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
+Bitcoin XT is an implementation of a Bitcoin Cash (BCH) full node that embraces Bitcoin's original vision of simple, reliable, low-cost transactions for everyone in the world. Bitcoin XT originated as a series of patches on top of Bitcoin Core and is now a independently maintained software fork. See our notable [features](https://bitcoinxt.software/patches.html).
 
-XT uses the same data directories as Core so you can easily switch back and forth. You will keep and continue updating the same block chain.
+Bitcoin XT downloads are code signed and are [built reproducibly using gitian](https://github.com/bitcoinxt/gitian.sigs). Discussion can be found at the [XT Gitter](https://gitter.im/bitcoinxt/Lobby).
+
+Data Directory Compatibility
+============================
+
+If using a data directory from a different BCH or BTC full node, specify `-reindex` on first launch (reason: UTXO database format).
```

Comment:
Having the same utxo format won’t help because we don’t do obfuscation. It would help _from_ XT though

Reply:
So, remove reason:, or update to list two reasons: you have to reindex?